### PR TITLE
Fix `inbound_liquidity_msats`

### PR DESCRIPTION
**Pull request by danielgranhao**
Addresses #1018

### CODE REVIEW

File: libs/sdk-core/src/models.rs [hunk 1]
```diff
@@ -629,6 +629,7 @@ pub struct NodeState {
     pub max_single_payment_amount_msat: u64,
     pub max_chan_reserve_msats: u64,
     pub connected_peers: Vec<String>,
+    pub max_receivable_single_payment_amount_msat: u64,
```
Comment (ok300) [inline libs/sdk-core/src/models.rs:632]: I would add some rustdocs about the meaning of the changed `NodeState` fields, for example

```rust
pub struct NodeState {
    ...

    /// Maximum receivable in a single payment (single channel)
    pub max_receivable_single_payment_amount_msat: u64,

    /// Total receivable on all available channels
    pub inbound_liquidity_msats: u64,
}
```

**Review by ok300 — APPROVED** [review]
Looks good, added just a NIT on comments.

File: libs/sdk-core/src/models.rs (hunk 1 above)
Comment (danielgranhao) [reply libs/sdk-core/src/models.rs:634]: Added the proposed docs in [e2d48a4](https://github.com/breez/breez-sdk/pull/1062/commits/e2d48a4f6a2f640877ea4b10c24fce1d4c8b019a)

File: libs/sdk-core/src/models.rs [hunk 2]
```diff
@@ -629,6 +629,11 @@ pub struct NodeState {
     pub max_single_payment_amount_msat: u64,
     pub max_chan_reserve_msats: u64,
     pub connected_peers: Vec<String>,
+
+    /// Maximum receivable in a single payment (single channel)
```
Comment (JssDWt) [inline libs/sdk-core/src/models.rs:633]: I know this line was suggested by @ok300, but a little nit here still.

```suggestion
    /// Maximum receivable in a single payment without requiring a new channel open.
```

**Review by JssDWt — COMMENTED** [review]
I just have one nit, looking good!

File: libs/sdk-bindings/src/breez_sdk.udl [hunk 3]
```diff
@@ -187,6 +187,7 @@ dictionary NodeState {
     u64 max_single_payment_amount_msat;
     u64 max_chan_reserve_msats;
     sequence<string> connected_peers;
+    u64 max_receivable_single_payment_amount_msat;
```
Comment (roeierez) [inline libs/sdk-bindings/src/breez_sdk.udl:190]: Now that we add this field the former one (inbound_liquidity_msats) changes its semantic. 
My concern is that developers will upgrade and their logic will break without them knowing about it.
Can we rename the `inbound_liquidity_msats` so developers will get a compilation error on upgrade which will force them to change the code?
We can also leave it with the old semantic (and mark it as deprecated) and add another field (total_inbound_liquidity_msat) with the unified liquidity.
Anything that will not let this change go under the radar.

Comment (danielgranhao) [reply libs/sdk-bindings/src/breez_sdk.udl:190]: That's a very good point @roeierez. I'll rename it to `total_inbound_liquidity_msat`.

**Review by roeierez — APPROVED** [review]

**Review by JssDWt — APPROVED** [review]
LGTM!

**Pull request merged**